testnet/stacks-node/src/run_loop/neon.rs

```diff
@@ -180,8 +180,18 @@ impl RunLoop {
             event_dispatcher.register_observer(observer);
         }
 
+        // use test genesis data if use_test_genesis_chainstate is true in config
+        // otherwise default to USE_TEST_GENESIS_CHAINSTATE const
+        let use_test_genesis_data = if let Some(use_test_genesis_chainstate) =
+            self.config.node.use_test_genesis_chainstate
+        {
+            use_test_genesis_chainstate
+        } else {
+            USE_TEST_GENESIS_CHAINSTATE
+        };
+
```

Comment:
Here's a code snippet from `testnet/stacks-node/src/node.rs`:
```let use_test_genesis_data = if config.burnchain.mode == "mocknet" {
            // When running in mocknet mode allow the small test genesis chainstate data to be enabled.
            // First check env var, then config file, then use default.
            if env::var("BLOCKSTACK_USE_TEST_GENESIS_CHAINSTATE") == Ok("1".to_string()) {
                true
            } else if let Some(use_test_genesis_chainstate) =
                config.node.use_test_genesis_chainstate
            {
                use_test_genesis_chainstate
            } else {
                USE_TEST_GENESIS_CHAINSTATE
            }
        } else {
            USE_TEST_GENESIS_CHAINSTATE
        };
```

I'm wondering if you'd possibly want to (a) check the env var value here, and/or (b) create a helper function to determine the value of the var `use_test_genesis_data` and use it in both places.